src/main/java/mate/project/exception/CustomGlobalExceptionHandler.java

```diff
@@ -0,0 +1,59 @@
+package mate.project.exception;
+
+import java.time.LocalDate;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.HttpStatusCode;
```

Comment:
HttpStatus is imported twice. Remove the import statement for org.springframework.http.HttpStatusCode as it is not required.